### PR TITLE
update to laravel 1st

### PR DESCRIPTION
**Pull request by vinhtruong94**

### CODE REVIEW

**Comment by psaunders** [comment]
https://github.com/IserveU/IserveU/issues/700

**Comment by vinhtruong94** [comment]
Changes in composer.json to upgrade to laravel 
- laravel framework from 5.3 to 5.4
- remove sofa Eloquence as the system does not use it.
- upgrading spatie/laravel-backup to v6.
- upgrading phpunit to 5.7 ( we use 5.4 before, they have released ^ 6.0 vers already, but 5.7 run well )
- added BrowserKitTesting from Taylor Rotwell ( to be able to maintain our 5.3 reliable phpunit testing system)
- added another folder on Test folder( we will have 2 test class, TestCase and BrowserKitTestCase, the latter is for old test system to be runnable, TestCase will be extended on new tests. That way we can write new tests with new syntax and keep our old tests only to update it later because Taylor may close BrowserKitTestCase ).
Changes in other files:
-mostly changes are "extend BrowserKitTestCase" instead of "extend TestCase".
-changes in method syntax ( new method names in Laravel 5.4).
https://laravel.com/docs/5.4/upgrade

**Comment by psaunders** [comment]
Waiting on this: https://github.com/tightenco/mailthief/issues/65

File: tests/unit/Jobs/Emails/PrepareMotionSummaryTest.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ public function motion_summary_email_contains_motions_that_recently_closed()
 
         dispatch(new PrepareMotionSummary());
 
-        Mail::assertSentTo([$user], MotionSummary::class, function ($mail) use ($motion) {
+        Mail::assertSent(MotionSummary::class, function ($mail) use ($motion) {
```
Comment (psaunders) [inline tests/unit/Jobs/Emails/PrepareMotionSummaryTest.php]: The replacement for "assertSentTo" is not "assertSent", you also need to now add a "to" check like the documentation.

Comment (psaunders) [inline tests/unit/Jobs/Emails/PrepareMotionSummaryTest.php]:
```
$mail->hasTo($user->email) &&
```

Comment (vinhtruong94) [inline tests/unit/Jobs/Emails/PrepareMotionSummaryTest.php]: Tests upgraded

**Pull request merged**